uptime_service_validation/coordinator/coordinator.py

```diff
@@ -100,6 +100,13 @@ def main():
                     prev_batch_end, cur_batch_end
                 )
             )
+
+            webhookURL = os.environ["WEBHOOK_URL"]
+            if (end - start < float(os.environ["ALARM_ZK_LOWER_LIMIT_SEC"])): 
+                sendSlackMessage(webhookURL, f'ZkApp Validation took {end- start} seconds, which is too quick', logging)
+            if (end - start > float(os.environ["ALARM_ZK_UPPER_LIMIT_SEC"])): 
+                sendSlackMessage(webhookURL, f'ZkApp Validation took {end- start} seconds, which is too long', logging)
```

Comment:
I think this functionality should be made optional (i.e. for testing we don't want it enabled). In particular `os.environ['WEBHOOK_URL']` will throw error when error when env var is not set. Better choice would be `os.environ.get` with some `if` statement checking the presence of env vars and based on that deciding if send should be made or not.

Also some logging here would be good.

Reply:
Thanks. We have logging in the function, but I'll put the if-statement in.